index-test.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require_once 'DeckDao.php';
```

Comment:
Make sure to get rid of any test files like this before submitting a pull request, we only want to main to have production ready code

Reply:
Okay will delete